src/webportal/src/app/job/job-submit/job-submit.component.js

```diff
@@ -28,6 +28,7 @@ const userAuth = require('../../user/user-auth/user-auth.component');
 const jobSchema = require('./job-submit.schema.js');
 const querystring = require('querystring');
 const stripJsonComments = require('strip-json-comments');
+const uuid = require('uuid');
```

Comment:
Have not seen the `uuid` included in `package.json`, and I don't think uuid is required here. In browser side, uuid just uses crypto level random number instead of `Math.random()` to avoid more collisions, since we have a directly response when user is submitting with the same job name, so I don't think avoid more collisions here is important.

Reply:
fixed